site/topics/car-rental/car-rental.rst

```diff
@@ -14,11 +14,37 @@ What have we seen so far?
 * Logic
 * If/Else
 
+* Each of the individual topics may feel simple on their own
+* The difficult part tends to be when putting these topics together to solve complex problem
+
+
+Writing Bigger Programs
+=======================
+
+* There is no single correct way to write programs, but there are some strategies
+* For now I recomment a bottom up, incremental approach
+
+    # Start with an empty function and have it return some arbitrary constant value (e.g., ``0``)
+    # Run the function and verify that it does what you expect
+    # Add one or two lines of code
+    # Run the function and verify that it does what you expect
+    # Repeat
+
+* This incremental strategy is great because a lot of the problem solving you will be doing will be incremental
+* ADdditionally, it helps you make sure everything is working along the way

```

Comment:
ADdditionally -> Additionally 

Some kind of grammar checker would be nice to have as part of the build.

Reply:
I think maybe in the last pycharm update some plugin stopped working because I swear it was flagging these before.